deepview_profile/commands/analysis.py

```diff
@@ -152,6 +158,10 @@ def actual_main(args):
         if args.energy_compute or is_return_all:
             data['analysisState']['energy'] = next_message_to_dict(energy_compute(session))
 
+        if not args.no_files:
+            path = os.path.dirname(args.entry_point)
+            data['encodedFiles'] = files_encoded_content(path)

```

Comment:
We should not include every python file in the directory, but rather the ones referred to in the breakdown information.

Reply:
Seems to be similar to the standards of the project